src/js/App.js

```diff
@@ -29,6 +29,7 @@ import Country from './Country';
 import NotFoundPage from './NotFoundPage';
 import News from './News';
 import Information from './Information';
+import HomePageSelector from './HomePageSelctor';
```

Comment:
Why is it imported here? I feel it should be in Home instead?

Reply:
Removed.